backend/lcfs/web/api/compliance_report/schema.py

```diff
@@ -114,6 +123,9 @@ class FinalSupplyEquipmentSchema(BaseSchema):
 
 class ComplianceReportBaseSchema(BaseSchema):
     compliance_report_id: int
+    compliance_report_group_uuid: Optional[str]
+    version: Optional[int]
+    supplemental_initiator: Optional[SupplementalInitiatorType]
```

Comment:
should be included `reporting_frequency`  as well?

Reply:
yes added ty.